airbyte-integrations/bases/connector-acceptance-test/connector_acceptance_test/config.py

```diff
@@ -132,11 +115,6 @@ class IgnoredFieldsConfiguration(BaseConfig):
     bypass_reason: Optional[str] = Field(default=None, description="Reason why this field is considered ignored.")
 
 
-ignored_fields: Optional[Mapping[str, List[IgnoredFieldsConfiguration]]] = Field(
```

Comment:
As it is not shared anymore for basic_read, the definition has been moved to https://github.com/airbytehq/airbyte/pull/35645/files#diff-6d30b0c7923582524ce6f15dccff0e22689713a162893c6dcd268c6af46f2302R194